### ISSUE TITLE
termux support

### ISSUE THREAD

**Issue by choman** [gravitational/teleconsole#48]
Sitting on my chromebook and loving termux.   Thought to myself, teleconsole would be awesome.  Can't determine the platform

No worries if this is tossed aside, it would be awesome though

Cheers, keep up the great work

**Comment by kontsevoy** [comment]
@choman you mean the `get` script doesn't work? Have you tried simply downloading the binary from [Github releases](https://github.com/gravitational/teleconsole/releases) for ARM?

**Comment by dmnsea** [comment]
@kontsevoy
>  ... binary for ARM?

No. It's just not possible to launch Teleconsole in Termux as regular binary (/path/to/teleconsole).
Termux's bash says:
`bash: ./teleconsole: No such file or directory`

I'm not sure where is the reason of such behaviour, cause some linux-arm binaries(other programs) works well in Termux, and other fails with same error "No such file or directory".

@choman , I think it may be something wrong on Termux or Android side (system), but not sure where is it exactly.
